picasso/interfaces/rest.py

```diff
@@ -106,7 +106,7 @@ def visualize():
         vis.update_settings(session['settings'])
     output = vis.make_visualization(
         inputs, output_dir=session['img_output_dir'])
-    return jsonify(output=output)
+    return jsonify(output[0])
```

Comment:
can you tell me why this changed?

Reply:
It changed because the API doesn't allow visualising more than one image per request anyways. If you want to visualise several images you'd need to request each separately. So, I just reduce the JSON list with one JSON object to just the JSON object. It makes the interface cleaner in my opinion and by doing it here I'm not breaking the current web front end.

Reply:
Sounds good!